extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -269,12 +341,23 @@ export class ConsentUI {
     }
 
     const data = getData(event);
-    if (!data) {
+    if (!data || !data['type']) {
       return;
     }
 
     if (data['type'] == 'consent-ui-ready') {
       this.iframeReady_.resolve();
     }
+
+    if (data['type'] === 'consent-ui-enter-fullscreen') {
+
+      if (!this.isIframeVisible_) {
+        return;
```

Comment:
Do you think we should tell the iframe if the `enter-fullscreen` succeed or not? Please add a TODO and we can address that once we refactor the message type.